README.md

```diff
@@ -1,3 +1,66 @@
 # apostrophe-seo
 
-Test.
+SEO for [ApostropheCMS](http://apostrophecms.org/).
+
+Add useful meta fields to `apostrophe-custom-pages`, `apostrophe-pieces`, and `apostrophe-global`.
+
+## Installation
+
+```bash
+npm install apostrophe-seo --save
+```
+
+## Use
+
+Configure `apostrophe-seo` in `app.js`.
+
+```js
+const apos = require('apostrophe')({
+  shortName: 'project',
+  modules: {
+    'apostrophe-seo': {}
+  }
+```
+
+Add to `outerLayout.html` in `apostrophe-templates/views/`.
+
+```nunjucks
+{% if data.piece %}
+  {% if data.piece.seoPageTitle %}
+    {% set title = data.piece.seoPageTitle %}
```

Comment:
"seoPageTitle" and friends include the redundant word "page". Either we already know it's a page, or it's a piece so it's just wrong. Instead use "seoTitle" and friends.

Reply:
See 7b96c3d816b3e1b58cd3f85a97961993dc7c2fdb